app/Providers/PrometheusServiceProvider.php

```diff
@@ -94,6 +94,21 @@ public function register() {
         Prometheus::addGauge("is_maintenance_mode_active")
                   ->helpText("Is the Laravel Maintenance Mode active right now?")
                   ->value($this->app->maintenanceMode()->active());
+
+        Prometheus::addGauge("oauth_tokens")
+                  ->helpText("How many users do the clients have?")
+                  ->labels(["app_name"])
+                  ->value(function() {
+                      return DB::table("oauth_access_tokens")
+                               ->join("oauth_clients", "oauth_access_tokens.client_id", "=", "oauth_clients.id")
+                               ->groupBy("oauth_clients.name")
```

Comment:
Additionally group by `oauth_access_tokens.user_id`?

Reply:
I've added "unique users per client" and "revoked tokens per client" as separate metrics: 25260255cedea070828ae9538a4bf5301e4fd3bb